admin/js/boldgrid-backup-admin-settings-autoupdate.js

```diff
@@ -66,51 +89,68 @@ BOLDGRID.SETTINGS = BOLDGRID.SETTINGS || {};
 				$pluginToggles = $bgBox.find( '.plugin-toggle' ),
 				$themeToggles = $bgBox.find( '.theme-toggle' ),
 				$pluginsDefault = $bgBox.find( '#toggle-default-plugins' ),
-				$themesDefault = $bgBox.find( '#toggle-default-themes' );
+				$themesDefault = $bgBox.find( '#toggle-default-themes' ),
+				$timelyUpdatesEnabled = $bgBox.find( '#timely-updates-enabled' ),
+				$timelyUpdatesDisabled = $bgBox.find( '#timely-updates-disabled' ),
+				$timelyUpdatesDays = $bgBox.find( '#timely-updates-days' );
 
 			// If the updates section is not in use, then just return.
-			if ( ! $pluginsDefault.data( 'toggles' ) ) {
-				return;
-			}
+			// if ( ! $pluginsDefault.data( 'toggles' ) ) {
```

Comment:
Is this meant to be commented out?

Reply:
I commented that out when debugging something and didn't uncomment it. fixed